demos/drawer/permanent-drawer-above-toolbar.html

```diff
@@ -73,11 +74,11 @@
     </style>
   </head>
   <body class="mdc-typography demo-body">
-    <nav class="mdc-permanent-drawer">
-      <div class="mdc-permanent-drawer__toolbar-spacer"></div>
+    <nav id="demo-theme-switch" class="mdc-drawer mdc-drawer--permanent">
+      <div class="mdc-drawer__toolbar-spacer"></div>
       <div class="mdc-list-group">
         <nav class="mdc-list">
-          <a class="mdc-list-item mdc-permanent-drawer--selected" href="#">
+          <a class="mdc-list-item mdc-drawer--selected" href="#">
```

Comment:
`mdc-drawer--selected` should be `mdc-list-item--selected`

Ditto in other files